src/Converter/Feature2Mesh.js

```diff
@@ -29,14 +30,16 @@ class FeatureMesh extends THREE.Group {
         this.meshesCollection.updateMatrix();
 
         this.crs = collection.crs;
+        this.#currentCrs = this.crs;
```

Comment:
In my opinion, it would make more sense to rename `this.crs` to something like `this.collectionCrs` and `this.#currentCrs` to `this.#crs`. WDYT ?

Also, I don't see the the current `this.crs` property being used outside of `FeatureMesh` class. Could it be set private ?

Reply:
I simplified the code